.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run a one-line script
+      run: echo Hello, world!
+    - name: Run a multi-line script
+      run: |
+        echo Add other actions to build,
```

Comment:
for (var adj of [
   'Raging',
   'Happy',
   'Sad',
]) {
    for (var noun of [
'Pharmacists',
'Hippo',
'Shark',
]) {
   print('The ' + adj + ' ' + noun
);
  }
}